sklearn/linear_model/passive_aggressive.py

```diff
@@ -36,6 +36,27 @@ class PassiveAggressiveClassifier(BaseSGDClassifier):
 
         .. versionadded:: 0.19
 
+    early_stopping : bool, default=False
+        Whether to use early stopping to terminate training when validation.
+        score is not improving. If set to True, it will automatically set aside
+        a fraction of training data as validation and terminate training when
+        validation score is not improving by at least tol for
+        n_iter_no_change consecutive epochs.
+
+        .. versionadded:: 0.20
+
+    n_iter_no_change : int, default=1
```

Comment:
I would put the default to 5 (or maybe even 10). Otherwise, the estimator runs the risk to stop way to early.

Reply:
I agree, but this would break user's code.

Reply:
Why? If it was hard-coded to 2, setting the default to 1 will also break compat.

Reply:
Actually, in master the effective patience is 1, my bad.

I think this is a really bad default value. We should at least issue a `FutureWarning` and say that in the next version it will be set to 5 (or 10).

Or because it can be considered a bug to stop way to early at random I would actually be in favor of changing the default right now to 5 (or 10) and mention that as a bugfix in the change log.